cocos/spine/track-entry-listeners.ts

```diff
@@ -38,4 +44,23 @@ export class TrackEntryListeners {
         }
         return entry.listener;
     }
+
+    static emitListener (id: number, entry: spine.TrackEntry, event: spine.Event): void {
+        const listener = TrackEntryListeners._listenerSet.get(id);
+        if (!listener) return;
+        const listener2 = listener as TrackListener2;
+        if (listener2) {
+            listener2(entry, event);
+        }
+    }
+
+    static addListener (listener: CommonTrackEntryListener): number {
+        const id = ++_listener_ID;
+        TrackEntryListeners._listenerSet.set(id, listener);
+        return id;
+    }
+
+    private static _listenerSet = new Map<number, CommonTrackEntryListener>();
 }
+
+globalThis.TrackEntryListeners = TrackEntryListeners;
```

Comment:
Why register to global variable, it's strongly recommended to avoid

Reply:
emmm... i need import a static function to c++ in WebAssembly JS glue code.
I don't find a suitable way to get the TrackEntryListeners

Reply:
Try to use `legacyCC.internal`